src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -2368,6 +2370,7 @@ def test_vmss_existing_lb(self, resource_group):
         self.cmd('network lb create -g {rg} -n {lb} --backend-pool-name test')
         self.cmd('vmss create -g {rg} -n {vmss} --load-balancer {lb} --image UbuntuLTS --admin-username clitester --admin-password TestTest12#$')
 
+    @unittest.skip('Can\'t test lb. Not allowed to open ports')
```

Comment:
Why skip it directly ? I'm OK to skip it in live test, however with this decorator, the recording mode will also be skipped.

Reply:
If can't live, then can't replay.

Reply:
Can't agree. I think daily live test is different with manual live test. Although this test can not be live run daily, I can still manually run it to generate a recording for it.

Reply:
We need to disable the policy to run it live and generate a recording for it. It reduces stability of testing results if policies are disabled and enabled frequently.

Reply:
So I mean there should be a decorate to skip live test only.